ui/src/pages/backoffice/Patron/PatronDetails/components/PatronPastLoans/PatronPastLoans.js

```diff
@@ -0,0 +1,122 @@
+import { dateFormatter } from '@api/date';
+import { DocumentTitle } from '@components/Document';
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { Loader, Error, ResultsTable } from '@components';
+import { loan as loanApi } from '@api';
+import { invenioConfig } from '@config';
+import { BackOfficeRoutes } from '@routes/urls';
+import { SeeAllButton } from '@pages/backoffice/components/buttons';
+
+export default class PatronPastLoans extends Component {
+  componentDidMount() {
+    const patronPid = this.props.patronDetails.user_pid
+      ? this.props.patronDetails.user_pid
+      : null;
```

Comment:
```suggestion
    const patronPid = _get(this.props, 'patronDetails.user_pid', null)
```

Reply:
👍 yeah I need to switch my head to force it  to use this